src/appengine/handlers/cron/ml_train.py

```diff
@@ -28,17 +28,25 @@ class Handler(base_handler.Handler):
   def get(self):
     """Handle a GET request."""
     for job in data_types.Job.query():
+
       if not utils.string_is_true(
           job.get_environment().get('USE_CORPUS_FOR_ML')):
         continue
 
+      if utils.string_is_true(job.get_environment().get('USE_GRADIENTFUZZ')):
+        task_name = 'gradientfuzz'
```

Comment:
Nit: could you rename each of these tasks to train_gradientfuzz and train_rnn_generator? That seems to clear up any ambiguity. Sorry to ask you to rename those files again.

Reply:
Yeah -- makes sense, given that training and generation are separate.